playbooks/ORCHESTRATION.md

```diff
@@ -0,0 +1,219 @@
+# Orchestration
+
+This playbook describes how to orchestrate a root signing event.
+
+## Pre-work
+
+1. Check the [calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing events. 
+
+2. Make updates to the targets and delegation configuration files, see [configuration](#targets-and-delegation-configuration).
+
+3. Double-check the configured [role expirations](https://github.com/sigstore/root-signing/blob/e3f1fe5e487984f525afc81ac77fa5ce39737d0f/cmd/tuf/app/init.go#L28).
+
+4. Set environment variables, including previous repository and online signer references (details [here](#key-configuration)). 
+
+| Variable      | Description | Example |
+| ----------- | ----------- | ----------- | 
+| GITHUB_USER      | The GitHub user, used to create PRs and commit messages       | asraa       |
+| BRANCH   | The working branch, in case of staging script or configuration changes.        | main        |
+| LOCAL   | If enabled, keeps git state dirty and does not create pull requests. Used to run root signing locally for testing.       |         |
+| REPO   | Specifies the repository folder, see [Configuration](#configuration)       |   `ceremony/2022-02-22`      |
```

Comment:
Should you also mention needing to set this if you're doing the signing across multiple days, since the date changes?

Reply:
These are all variables that need to be set unless indicated, so I think that's clear? I did add that by default it chooses the current date.

This might not even be a ceremony/date repo -- could also be `repository/repository`.